lib/lib.nix

```diff
@@ -84,7 +125,7 @@
     };
 
     lockfilePath = "${sources}/nickel.lock.ncl";
-    expectedLockfileContents = organistLib.buildLockFileContents lockFileContents;
+    expectedLockfileContents = buildLockFileContents lockFileContents;
```

Comment:
Since we don't have this dependency on `organistLib` any more, can you drop it from the arguments of this file altogether? That'll make the whole thing a bit simpler